CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Dependencies
 Bump gl-native to v10.8.0-beta.1, common to v23.0.0-beta.1. ([1568](https://github.com/mapbox/mapbox-maps-android/pull/1568))
+* Fix ANR while open and close MapView quickly with textureView set to true, fix potential crash on RenderHandlerThread creation. ([1567](https://github.com/mapbox/mapbox-maps-android/pull/1567))
```

Comment:
this should go to `##Bug fixes` section above; 
also I'd rephrase it as this ANR not truly related to texture view - perhaps something like 
```suggestion
* Fix possible ANR when destroying renderer. ([1567](https://github.com/mapbox/mapbox-maps-android/pull/1567))
```